apps/ejabberd/src/mod_mam_muc.erl

```diff
@@ -325,22 +323,22 @@ is_room_action_allowed_by_default(Action, From, To) ->
     end.
 
 
--spec is_room_owner(From :: ejabberd:jid(), To :: ejabberd:jid()) -> boolean().
-is_room_owner(From, To) ->
-    ejabberd_hooks:run_fold(is_muc_room_owner, To#jid.lserver, false, [To, From]).
+-spec is_room_owner(User :: ejabberd:jid(), Room :: ejabberd:jid()) -> boolean().
+is_room_owner(User, Room) ->
+    ejabberd_hooks:run_fold(is_muc_room_owner, Room#jid.lserver, false, [Room, User]).
 
 
 %% @doc Return true if user element should be removed from results
--spec is_user_identity_hidden(From :: ejabberd:jid(), ArcJID :: ejabberd:jid()) -> boolean().
-is_user_identity_hidden(From, ArcJID) ->
-    case mod_muc_room:can_access_identity(ArcJID, From) of
-        {error, _} -> true;
-        {ok, CanAccess} -> (not CanAccess)
+-spec is_user_identity_hidden(User :: ejabberd:jid(), Room :: ejabberd:jid()) -> boolean().
+is_user_identity_hidden(User, Room) ->
+    case ejabberd_hooks:run_fold(can_access_identity, Room#jid.lserver, false, [Room, User]) of
```

Comment:
Maybe it'd be good to prefix this hook name with sth indicating this relates to MUC(Light)?